### PR TITLE
doc(release): Release notes for Cloud 24.07

### PR DESCRIPTION
**Pull request by pkippes**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [x] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: cloud/releases/cloud-release-notes.md [hunk 1]
```diff
@@ -3,6 +3,119 @@ id: cloud-release-notes
...
+Release date: `August 06 , 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [BAM] Fixed Deprecation due to Symfony Upgrade.
+
+</details>
+
+
```
Comment (nmagniezCentreon) [inline cloud/releases/cloud-release-notes.md:118]: ## August xx, 2024

### Open Access to Centreon APIs

It is now possible to interact with Centreon through APIs and therefore integrate Centreon with any monitoring or data processing tool you may use, automatize monitoring configuration updates and any other use case you may have in mind. Access to APIs is granted thanks to tokens linked to contacts for access right management, but completely independent in terms of lifecycle.
API documentation is available here (link)

### Centreon Dashboards

It is now possible to declare dashboards playlists as public. An URL is genrated and give access to the dashboard playlist without authentication. This feature is ideal to display dahboards on big screens.
Note that this feature was implemented in a completely secured way. No confidential information is carried by the URL, nor it is possible to have access to any other information or data than the one displayed.
Note: MAP widget is not supported yet on public playlists.

This version of Centreon Dashboards comes with many other improvements. 
Graph widget allows much more configuration (automatic curves color adjustement for metric of same kind, bar chart and stacked bar chart presentation, 3 different curves design, and many others).
Widgets are also now more compact, allowing to display more of them on a single display.
And finally, freshness of data is now exposed and user can reload data at the widget level.

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

File: cloud/releases/cloud-release-notes.md [hunk 2]
```diff
@@ -3,92 +3,130 @@ id: cloud-release-notes
...
+**Miscellaneous improvements**
+
+  Resource Status filter modal has been improved by adding resource type and host name filters.
+
+### Centreon Business Edition
+
+**MAP**
+
+  The Geoview modal has been redesigned.
+In standard maps, it is now possible to hide container labels and to display status as background color for resources in weather style.
```
Comment (cg-tw) [inline cloud/releases/cloud-release-notes.md:35]:
```suggestion
In standard maps, it is now possible to hide container labels and to use the status as background color for resources in weather style.
```

File: cloud/releases/cloud-release-notes.md [hunk 3]
```diff
@@ -3,92 +3,130 @@ id: cloud-release-notes
...
+
 ## June 26, 2024
 
-### Resource Access Management
+### Centreon IT Edition
+
+**Resource Access Management**
 
-Resource Access Management allows administrators to assign specific access permissions to each user or user group. This enables control over which users can access particular monitored resources. An intuitive interface helps manage these permissions, while audit logs provide a record of access activities. This feature supports organizations in maintaining security and efficiency by allowing tailored access control.
+  Resource Access Management allows administrators to assign specific access permissions to each user or user group. This enables control over which users can access particular monitored resources. An intuitive interface helps manage these permissions, while audit logs provide a record of access activities. This feature supports organizations in maintaining security and efficiency by allowing tailored access control.
```
Comment (cg-tw) [inline cloud/releases/cloud-release-notes.md:44]:
```suggestion
Resource Access Management allows administrators to assign specific access permissions to each user or user group. This enables control over which users can access particular monitored resources. An intuitive interface helps manage these permissions, while audit logs provide a record of access activities. This feature supports organizations in maintaining security and efficiency by allowing tailored access control.
```

File: cloud/releases/cloud-release-notes.md [hunk 4]
```diff
@@ -3,36 +3,71 @@ id: cloud-release-notes
 title: Centreon Cloud release notes
 ---
 
+## August xx, 2024
```
Comment (cg-tw) [inline cloud/releases/cloud-release-notes.md:6]:
```suggestion
## August 22, 2024
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md [hunk 5]
```diff
@@ -2,35 +2,72 @@
 id: cloud-release-notes
 title: Centreon Cloud - notes de release
 ---
+
+## August xx, 2024
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md:6]:
```suggestion
## August 22, 2024
```

**Pull request merged**